README.md

```
@@ -50,10 +56,21 @@
+### Local App setup for CLI
 
-- Set up your local environment for React Native following the environment setup guide.
-- Clone the mobile app repository: git clone https://github.com/Real-Dev-Squad/mobile-app.git
+- Set up your local environment for React Native CLI following the environment setup guide.
+- Clone the mobile app repository from develop branch: git clone https://github.com/Real-Dev-Squad/mobile-app.git
+- Run the command: `yarn`
+- Start contributing!
+
+### Local App setup for Expo
+
+- Set up your local environment for React Native Expo following the environment setup guide.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)